data/abilities.js

```diff
@@ -1089,6 +1093,7 @@ exports.BattleAbilities = {
 		onResidual: function (pokemon) {
 			if (pokemon.status && this.isWeather(['raindance', 'primordialsea'])) {
 				this.debug('hydration');
+				this.add('-activate', pokemon, 'ability: Hydration');
```

Comment:
>if an ability activation is a standalone message, protocol should use an `-ability` tag

Is there actually any rhyme or reason to when `-ability` is used over `-activate`? The only relationship between the current list of things that use `-ability` that I saw was they all activate on switch-in, except Sturdy.

I agree that sending it with `-curestatus` would be better, but I also think this is okay for now. There are a number of other Abilities I left out of this PR because I haven't thought of any appropriate solutions yet. (Limber and friends on attempted status, Static and friends on hit, etc.)